content/rs/administering/designing-production/hardware-requirements.md

```diff
@@ -31,9 +31,9 @@ requirements for a production system:
+| # of nodes per cluster  | At least 3 nodes (2 data nodes and 1 quorum node) are needed in order to support a reliable, highly available deployment that handles process failure, node failure, and network split events in a consistent manner. | 3 nodes | >=3 nodes; must be odd number of nodes to maintain quorum |
 | # of cores per node | RS is based on a multi-tenant architecture and can run multiple Redis processes (or shards) on the same core without significant performance degradation.</br>Once the CPU load of a node has reached a certain threshold, RS tries to migrate "noisy" shards to a different node in the cluster and sends an alert to the operator.</br>If your application is designed to impose a lot of load on your Redis database, make sure that you have at least one available core for each shard of your database.</br>Additional recommendations:</br>1. If some of the cluster nodes are utilizing more than 80% of the CPU, you should look at migrating busy resources to less busy nodes.</br>2. If all the cluster nodes are utilizing over 80% of the CPU, scale-out the cluster by [adding a node]({{< relref "/rs/administering/cluster-operations/adding-node.md" >}}). | 4 cores | >=8 cores |
-| RAM | Defining your RAM size should be part of the capacity planning for your Redis usage. Since Redis uses a relatively large amount of buffers (i.e. for slave communication, for the client communication and for pub/sub commands) the operator should take extra care to maintain at least 30% of the RAM "unused" on each node.</br>Additional recommendations:</br>1. If some of the cluster nodes are utilizing more than 65% of the RAM, you should look at migrating busy resources to less busy nodes.</br>2. If all the cluster nodes are utilizing over 70% of the RAM, you should look to scale out the cluster by [adding a node]({{< relref "/rs/administering/cluster-operations/adding-node.md" >}}).</br>3. Do not run other memory-consuming systems on the same machine that is used as an RS node. | 15GB | >=30GB |
+| RAM per node | Defining your RAM size should be part of the capacity planning for your Redis usage. Since Redis uses a relatively large amount of buffers (i.e. for slave communication, for the client communication and for pub/sub commands) the operator should take extra care to maintain at least 30% of the RAM "unused" on each node.</br>Additional recommendations:</br>1. If some of the cluster nodes are utilizing more than 65% of the RAM, you should look at migrating busy resources to less busy nodes.</br>2. If all the cluster nodes are utilizing over 70% of the RAM, you should look to scale out the cluster by [adding a node]({{< relref "/rs/administering/cluster-operations/adding-node.md" >}}).</br>3. Do not run other memory-consuming systems on the same machine that is used as an RS node. | 15GB | >=30GB |
```

Comment:
@bmansheim - the change is only the addition of "per node"? 
Should we do the same also for "cores"